cgroups/cgroups.go

```diff
@@ -34,6 +34,9 @@ type Manager interface {
 
 	// Set the cgroup as configured.
 	Set(container *configs.Config) error
+
+	// Enters the specified process into these cgroups.
```

Comment:
nit: Rephrase as `Enters the specified process into the cgroup set` just to be consistent.